temporalio/client.py

```diff
@@ -445,6 +450,9 @@ async def start_workflow(
             rpc_metadata: Headers used on the RPC call. Keys here override
                 client-level RPC metadata keys.
             rpc_timeout: Optional RPC deadline to set for the RPC call.
+            eager_start: Reduce the latency to start this workflow.
+                It requires that the client is shared between the worker and the starter.
+                This is currently experimental.
```

Comment:
```suggestion
            eager_start: Potentially reduce the latency to start this workflow by
                encouraging the server to start it on a local worker running with
                this same client.
                This is currently experimental.
```

Pedantic, don't have to change

Reply:
Much clear, thanks!